processing/data_collection/gazette/spiders/mg_contagem.py

```diff
@@ -0,0 +1,48 @@
+from dateparser import parse
+import datetime as dt
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgContagemSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3118601"
+    name = "mg_contagem"
+    allowed_domains = ["contagem.mg.gov.br"]
+    start_urls = ["http://www.contagem.mg.gov.br/?se=doc"]
+
+    def parse(self, response):
+        """
+        @url http://www.contagem.mg.gov.br/?se=doc&pagina=2
+        @returns items 15 15
+        @scrapes date file_urls is_extra_edition territory_id power scraped_at
+        """
+        anchor_elements = response.css(".texto11pt a")
+
+        urls = [
+            response.urljoin(url)
+            for url in anchor_elements.css("::attr(href)").re(".+pdf")
+        ]
+        extra_editions = ["complementar" in url for url in urls]
+
+        dates_in_sentence = anchor_elements.css("p span:last-child ::text").re(
+            "(\d{1,2}\s+de\s+\w+\s+de\s+\d{4})"
+        )
+        dates = [parse(date, languages=["pt"]).date() for date in dates_in_sentence]
```

Comment:
It's not a major issue. But I would prefer import `parse` function from the `datetimeparser` with a different name. Just to not confuse it with the `parse` method from the spider. Just to improve readability.

Reply:
Sorry for taking all this time to fix something so simple, but now it is ready :rocket: